audbackend/core/backend.py

```diff
@@ -399,7 +408,8 @@ def put_archive(
             verbose: show debug messages
 
         Raises:
-            FileNotFoundError: if one or more files do not exist
+            BackendError: if an error is raised on the backend
+            FileNotFoundError: if one or more ``files`` do not exist
             FileNotFoundError: if ``tmp_root`` does not exist
```

Comment:
Maybe I made a mistake here. I guess it should be ``src_root`` and not ``tmp_root``? At least there is no test for missing ``tmp_root``.

Reply:
Would you prefer three single entries for `FileNotFoundError`:

```
            FileNotFoundError: if ``src_root`` does not exist
            FileNotFoundError: if one or more ``files`` do not exist
            FileNotFoundError: if ``tmp_root`` does not exist       
```

or should we combine them:

```
            FileNotFoundError: if ``src_root``, ``tmp_root``, or one or more ``files`` do not exist
```

Reply:
For now I changed to:

![image](https://user-images.githubusercontent.com/10383417/232755218-870cbf8e-0fed-41b2-a036-8a79271e6bb2.png)

Reply:
Your solution looks fine to me.